integration_test/scenarios/app_grid_scenario.dart

```diff
@@ -34,7 +34,8 @@ class AppGridScenario extends BaseTestScenario {
     await Future.delayed(const Duration(seconds: 2));
 
     if (PlatformInfo.isAndroid) {
-      await $.native.pressBack();
+      await $.native.pressHome();
+      await $.native.openApp();
```

Comment:
why do we need reopen app? Command `await $.native.pressBack();` is not back to previous app